### PR TITLE
Feat/fallback component

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
Partially #2663

## Description
- Reworked fallback behavior and created separate component for it.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by rustem-nasyrov** [comment]
@aluarius, @m0ksem, please take a look at the preset and vuestic configuration behavior.

File: packages/ui/src/components/va-avatar/VaAvatar.vue [hunk 1]
```diff
@@ -73,16 +84,6 @@ export default defineComponent({
 
     const hasLoadError = ref(false)
 
-    const srcComputed = computed(() => {
-      if (props.src && props.fallbackSrc && hasLoadError.value) {
-        emit('fallback')
```
Comment (m0ksem) [inline packages/ui/src/components/va-avatar/VaAvatar.vue:78]: Might be helpful to emit fallback event on image, so user can react if image wasn't loaded.

File: packages/ui/src/components/va-avatar/VaAvatar.vue [hunk 2]
```diff
@@ -13,11 +13,12 @@
...
     <slot v-bind="avatarOptions" v-else>
       <img
-        v-if="srcComputed"
-        :src="srcComputed"
+        v-if="$props.src && !hasLoadError"
+        :src="$props.src"
         :alt="$props.alt"
         @error="onLoadError"
       >
+      <va-fallback v-else-if="hasLoadError && $props.src" v-bind="VaFallbackProps" />
```
Comment (m0ksem) [inline packages/ui/src/components/va-avatar/VaAvatar.vue:21]: Would be nice to have a slot for fallback maybe.

**Review by m0ksem — APPROVED** [review]
In my opinion, these props are good for global configuration, but if user wants to configure some specific image to have a fallback it is much easier to provide a slot. But it is not a big deal.

**Review by aluarius — APPROVED** [review]
We won't make a documentation page for this component?

**Comment by rustem-nasyrov** [comment]
> We won't make a documentation page for this component?

No, it's not that necessary component.

**Pull request merged**